### PR TITLE
Narrator was not able to read command output.

### PR DESCRIPTION
**Pull request by patelp3989**

### CODE REVIEW

**Review by ritikaguptams — DISMISSED** [review]

File: src/App/Resources/en-US/Resources.resw [hunk 1]
```diff
@@ -492,6 +492,11 @@ On retry</value>
   <data name="OutputConst.Text" xml:space="preserve">
     <value>Console Output:</value>
   </data>
+  </data>
+  <data name="OutputStack.Text" xml:space="preserve">
```
Comment (ritikaguptams) [inline src/App/Resources/en-US/Resources.resw:496]: Do we really need this ? Wouldn't it be binding by 'AutomationProperties.LabeledBy="{Binding ElementName=OutputConst}"' the above mentioned text ?

File: src/App/ConsolePage.xaml [hunk 2]
```diff
@@ -36,11 +37,12 @@
...
         </StackPanel>
         <TextBlock x:Name="OutputConst" Grid.Row="4" FontWeight="Bold" VerticalAlignment="Bottom" Padding="5,5,0,5" HorizontalAlignment="Left" x:Uid="OutputConst"/>
-        <Button x:Name="LaunchRD" x:Uid="LaunchRD" Click="LaunchRD_Click" Grid.Row="4" HorizontalAlignment="Center" Visibility="Collapsed"/>
-        <Button x:Name="ClearButton" x:Uid="ClearButton" Click="ClearButton_Click" Grid.Row="4" HorizontalAlignment="Right"/>
-        <ScrollViewer x:Name="ScrollView" HorizontalScrollBarVisibility="Auto" Grid.Row="6">
-            <StackPanel x:Name="OutputStack" Margin="5" VerticalAlignment="Top" Orientation="Vertical" SizeChanged="OutputStack_SizeChanged"/>
+
+        <ScrollViewer x:Name="ScrollView" HorizontalAlignment="Stretch" HorizontalScrollBarVisibility="Auto" Grid.Row="6">
+            <TextBox  x:Name="OutputStack" AutomationProperties.LabeledBy="{Binding ElementName=OutputConst}" TextWrapping="NoWrap" IsReadOnly="True" AcceptsReturn="True" ScrollViewer.VerticalScrollBarVisibility="Auto"/>
```
Comment (nick5616) [inline src/App/ConsolePage.xaml:42]: Do we want text wrapping disabled?

File: src/App/ConsolePage.xaml.cs [hunk 3]
```diff
@@ -520,7 +507,5 @@ public void Dispose()
         private int _cmdHistoryIndex;
         private string _containerIp;
         private const int MaxCmdHistory = 100;
-        private const int MaxBlocks = 10; // @500 lines per block this is 5000 lines or 10 commands maximum
-        private const int MaxLinesPerBlock = 500;
```
Comment (nick5616) [inline src/App/ConsolePage.xaml.cs:524]: Is a code-defined max size no longer necessary now that we're using a textbox? How much data can a textbox handle?

**Review by nick5616 — COMMENTED** [review]
This may fix one of bugs as well :)

File: src/App/ConsolePage.xaml.cs (hunk 3 above)
Comment (patelp3989) [reply src/App/ConsolePage.xaml.cs:524]: It was to store 10 command outputs before. I am currently only showing output for 1 command.

**Review by nick5616 — APPROVED** [review]

**Pull request merged**